quotes/data.json

```diff
@@ -190,12 +190,23 @@
         "CJDaBomb": {
             "discriminator": "0069",
             "avatar": "https://cdn.discordapp.com/avatars/777227436429017118/685b9d16041f82c53bd8d9d6c5ef24ab.png",
+            "nickname": "Slurpee™️",
             "roles": []
         },
         "Citrine": {
             "discriminator": "1282",
             "avatar": "https://cdn.discordapp.com/avatars/227118001206329344/ccd6eddd6a0e97fbfea9a6c7f2177fbf.png",
             "roles": []
+        },
+        "Max Hyper": {
+            "discriminator": "0310",
+            "avatar": "https://cdn.discordapp.com/avatars/217760439465934848/f42e3f3987f77bc74d0ff7eb05ed0b81.png",
```

Comment:
hmm actually this should prolly have `?size=128` on the end of it (maybe for lord samosa too)